pom.xml

```diff
@@ -55,7 +55,8 @@
     <!-- We are ignoring java doc warnings - this is because we are using JDK 11. Ideally we should not do that. -->
     <doclint>none</doclint>
     <sonar.analyzer.commons>1.25.0.1003</sonar.analyzer.commons>
-    <sonar.version>9.3.0.51899</sonar.version>
+    <sonar.version>9.8.0.203</sonar.version>
+    <sonar.api.impl.version>9.6.0.57076</sonar.api.impl.version>
```

Comment:
I had to use a different version for the implementation since it was not released in sync with the api. See the question from: https://discuss.sonarsource.com/t/sonar-plugin-api-relocated-in-maven-central/11143/3?u=costin.zaharia

Reply:
The reason why I asked is that I don't understand why we need to reference `impl`, and I'd prefer to clarify this before making changes to the repo.

If we don't have a functional need, I suggest to clarify internally. I didn't see somewhere that this is how we should consume the APIs from now on.

Reply:
We discussed also offline. The `impl` library is used only for tests and it has already the `<scope>test</scope>` property set.

Reply:
ah you're right

Reply:
sorry for the noise

Reply:
just wanted to confirm that many plugins use the `impl` package (in SonarQube) to have access to default implementations and other resources used for tests. Perhaps one day the API itself could have its own test resources but that's the current situation.